packages/next-app/src/app/internalmanager/[network]/[operationKind]/[tokenAddress]/page.tsx

```diff
@@ -39,16 +40,22 @@ import {
 import { refetchRequest } from "#/utils/fetcher";
 import { ArrElement, GetDeepProp } from "#/utils/getTypes";
 
+type TokenData = Omit<
+  ArrElement<GetDeepProp<SingleInternalBalanceQuery, "userInternalBalances">>,
+  "__typename"
+>;
```

Comment:
`__typename` shouldn't be added to responses unless we're explicitly querying for it. Is this not what you've observed?

Reply:
Actually no, the __typename is always there on the responses and is also expected from the type created by gql. 

In case of only having the token on the wallet we are creating the TokenData from wagmi info instead of gql and that's why there is no `__typename`